manifests/charts/base/templates/crd-all.gen.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.base.enableCRDs }}
```

Comment:
Why did this move? We already have these enabled in the templates optionally: https://github.com/istio/istio/blob/92a344ff7e2ad76e1e2f9dcd19f27c0ae7baf994/manifests/charts/base/templates/crds.yaml#L1

This change will break helm support

Reply:
Is this `enableCRDTemplates` still used?  thinks it is invalid now..  If it is still valid, how should I use it? because currently helm.go will install all crds under /crds under a chart.  So this flag is useless if we put crd-all.gen.yaml under crds.